stable/traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.30.0
+version: 1.33.0
```

Comment:
Not sure why you skip two versions?

Reply:
@timstoop because my other two PRs have 1.31 and 1.32. I couldn’t find any documentation on how to set versions.

Reply:
Don't assume merge order. Best to increment as if this is the only Traefik PR in the works right now and then let whoever is doing merges suggest conflict resolution to you if something else gets merged first.